code/modules/surgery/cavity_implant.dm

```diff
@@ -19,6 +19,7 @@
 	preop_sound = 'sound/surgery/organ1.ogg'
 	success_sound = 'sound/surgery/organ2.ogg'
 	var/obj/item/item_for_cavity
+	var/list/heavy_weight_whitelist = list(/obj/item/transfer_valve = TRUE)
```

Comment:
will repost: this isn't a typecache, we (I assume) want to catch subtypes here, so you should pass this through a GLOB, likely as  part of a `GLOBAL_LIST_INIT()` macro instead of as a local var